packages/app/ui/buttons/RadioButton.tsx

```diff
@@ -40,24 +40,23 @@ export function RadioButton({
   value,
   checked,
   name,
-  onChange,
+  onChange
 }: RadioButtonProps) {
   return (
-    <div className="flex">
+    <div className={twMerge("flex", className)}>
       <input
         type="radio"
         value={value}
         id={id}
         onChange={onChange}
         checked={checked}
         name={name}
-        className="w-0 h-0"
+        className="hidden"
```

Comment:
Hidden components aren't navigable with keyboard. So that's why they have 0 width and height. It's good to have components respecting ARIA. We don't know when someone will need this :)

Reply:
As per our discussion offline, we will dismiss this comment for now, as Firefox still displays the radio button bullets with `w-0 h-0` classes applied.

Visual evidence:
![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/94f8c75a-c266-418b-8bd1-be473826b4e0)

Reply:
Let's keep with hidden :)